tests/eval_files/trio911.py

```diff
@@ -742,6 +748,10 @@ async def foo_loop_static():
         await foo()
     yield  # error: 4, "yield", Stmt("yield", line-4)
 
+    for _ in """""":
+        await foo()
+    yield  # error: 4, "yield", Stmt("yield", line-4)
+
```

Comment:
cst's way of treating literal strings includes what the string quotation marks are, so added this test.